tests/Api/Admin/OrdersTest.php

```diff
@@ -65,4 +72,98 @@ public function it_gets_orders_for_customer(): void
             Response::HTTP_OK,
         );
     }
+
+    /** @test */
+    public function it_updates_an_order_billing_data(): void
```

Comment:
If it's not a convention to use `_data`, then `_address` is more relevant.

```suggestion
    public function it_updates_an_order_billing_address(): void
```